proto/quicksilver/airdrop/v1/proposals.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package quicksilver.airdrop.v1;
+
+import "gogoproto/gogo.proto";
+import "quicksilver/airdrop/v1/airdrop.proto";
+
+option go_package = "github.com/ingenuity-build/quicksilver/x/airdrop/types";
+
+message RegisterZoneDropProposal {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+    option (gogoproto.goproto_stringer) = false;
+
+    string title = 1;
+    string description = 2;
+
+    ZoneDrop zone_drop = 3 [ (gogoproto.moretags) = "yaml:\"zone_drop\"" ];
+    repeated ClaimRecord claim_records = 4 [ (gogoproto.moretags) = "yaml:\"claim_records\"" ];
```

Comment:
If we leave this as is, it means all the claim records must be submitted as json (making the payload far larger than it needs to be). If claim_records is a byte array, we can compress / optimise this payload to our heart's content, and expand this once the tx is in the application - thoughts?

Reply:
It is possible that the unpacking might be more costly in gas that the huge tx. But we ought to try this out.

Reply:
I agree, we should have the proposal payload as []byte (containing both ZoneDrop & []ClaimRecord); This will correspond with RegisterZoneProposals which are once-off type transactions, decompression overhead feels like the better option here vs massive uncompressed json blob.